source/ocsp-support/ocsp-support.rst

```diff
@@ -3,14 +3,14 @@ OCSP Support
 ============
 
 :Spec Title: OCSP Support
-:Spec Version: 1.3.1
+:Spec Version: 2.0.0
```

Comment:
I feel like changing whether OCSP is enabled by default is a breaking change and thus requires a major version bump.